crates/sqlexec/src/planner/dispatch.rs

```diff
@@ -649,6 +659,19 @@ impl<'a> SystemTableDispatcher<'a> {
             datasource.append_value(table.options.as_str());
         }
 
+        // Append temporary tables.
+        for table in self.ctx.list_temp_tables() {
+            // TODO: Assign OID to temporary tables
```

Comment:
I don't know if we'll want or need to assign oids to these tables. Going to check out what some other systems do.

Reply:
Yep, seems like duckdb does too. I think we go with 0 for now. We'll probably want to extend the `SessionCatalog` eventually to support the temp stuff.